docs/monthly/2023-06-monthly.md

```diff
@@ -0,0 +1,204 @@
+  - The aforementioned [End-to-end benchmarks](/benchmarks/end-to-end-benchmarks),
+  - [Ledger Micro-benchmarks](/benchmarks/ledger) as a comparison basis.
+
+#### Operating hydra nodes
+
+As a "dogfooding" exercise, the Hydra team is operating a Hydra Head
+on mainnet on top of which is running our `hydraw` application. We
+have observed the Head protocol got stuck a couple of times and
+post-mortem investigations lead us to a few improvements in how to
+operate a hydra-node and network:
+
+- Better reporting on the version of executable `hydra-node` and
+  `hydra-tui` which now report the Git SHA1 at which they were built
+- Reduction in the volume of logs emitted by hydra-node by removing
+  some chatty network-related logs and using ids to link `Begin/End`
+  pairs
+- Rework the
+  [log-filter](https://github.com/input-output-hk/hydra/blob/35f2964ba6d4a780a5f8e669f1afce565a492cec/hydra-cluster/exe/log-filter/Main.hs#L34)
+  tool to provide timings for various events and effects keyed by
+  transactions
+
+## Community
+
+#### Hydra hackathon / workshop
+
+The Hydra team is considering holding a workshop at/around Rare Evo at the end
+of August and is in contact with the event teams at IOG and the Rare Evo
+organizers. The concrete format, scope and agenda is still a bit unclear as we
+are contemplating whether to do a workshop/tutorial style or rather a
+introduction + challenge setting event. If it does not work out for Rare Evo, we
+will find another event or do it ourselves.
+
+If you are reading this and would be interested in joining such an event please
+drop us a line on [discord](https://discord.gg/Qq5vNTg9PT) or DM
+[@ch1bo](https://twitter.com/ch1bo_)! Ideally along with some thoughts on
+preferred format or what you would be interested in.
+
+#### Hydra for Auctions contributions and closing of project
+
+One of the Hydra lighthouse projects is slowly coming to an end. The
+collaboration project between IOG and MLabs on using Hydra for auctions is
+currently finalizing documentation and creating demonstration running the whole
+thing on a public testnet. Although the demo video was not available at the copy
+deadline of this report, we are confident to be showing more about this next
+month.
+
+The project yielded multiple github issues containing ideas and sparked great
+discussions on improving Hydra like [reported last
+month](https://hydra.family/head-protocol/monthly/2023-05#hydrozoa-850). It is
+also the first project which demonstrates how to move smart contracts from the
+layer 1 (L1) to the layer 2 (L2)! Overall it is a great example of establishing
+crucial state on L1 and achieving scalability through Hydra as L2.
+
+The code is fully open source and available on Github
+[hydra-auction](https://github.com/mlabs-haskell/hydra-auction/).
+
+## Conclusion
+
+The monthly review meeting for May was held on 2023-06-21 via Google Meet with
+these [slides][slides] and here is the [recording][recording].
+
```

Comment:
Again, left the first-person references in because I think it adds to the message.